docs/kubernetes/pouch_with_kuberneters_deploying.md

```diff
@@ -0,0 +1,152 @@
+# Pouch with kuberneters deploying
+
+- [Pouch deploying](#pouch-with-kuberneters-deploying)
+    - [Overview](#overview)
+    - [Kubernetes cluster](#kubernetes-cluster)
+        - [Install pouch](#install-pouch)
+        - [Install CNI](#install-cni)
+        - [Install kubelet](#install-kubelet)
+        - [Setting up the master node](#setting-up-the-master-node)
+        - [Setting up the worker nodes](#setting-up-the-worker-nodes)
+    - [Examples](#examples)
+
+
+## Overview
+
+This document shows how to easily install a kubernetes cluster with pouch runtime.
+
+## Kubernetes cluster
+
+#### Install pouch
+
+You can easily setup a basic Pouch environment, see [INSTALLATION.md](../../INSTALLATION.md).
+
+#### Install CNI
+
+On Ubuntu 16.04+:
+
+```
+apt-get update && apt-get install -y apt-transport-https
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+apt-get update
+apt-get install -y kubernetes-cni
+```
+
+On CentOS 7:
+
+```
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+       https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+setenforce 0
+yum install -y kubernetes-cni
+```
+
+#### Install kubelet
+
+On Ubuntu 16.04+:
+
+```
+apt-get install -y kubelet kubeadm kubectl
+```
+
+On CentOS 7:
+
+```
+yum install -y kubelet kubeadm kubectl
+```
+
+Configure kubelet with pouch runtime:
+
+```
+sed -i '2 i\Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=unix:///var/run/pouchcri.sock --image-service-endpoint=unix:///var/run/pouchcri.sock"' /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+systemctl daemon-reload
+systemctl restart kubelet
+```
+
+For more detail,please check [install kubelet](https://kubernetes.io/docs/setup/independent/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl).
```

Comment:
s/For more detail,please/For more details, please/

Add an s and a white space.